zulipterminal/ui_tools/color.py

```diff
@@ -0,0 +1,13 @@
+from re import match, ASCII
```

Comment:
Thanks @alex-ozdemir for working on this! I don't think we need to create a new file here. Just moving it to helper.py should be fine.